crmsh/sbd.py

```diff
@@ -161,10 +168,12 @@ def _update_configuration(self):
         """
         shutil.copyfile(self.SYSCONFIG_SBD_TEMPLATE, SYSCONFIG_SBD)
         self._determine_sbd_watchdog_timeout()
+        if utils.detect_virt() and self._sbd_devices:
+            self._delay_start = True
```

Comment:
BTW, given that Emil's PR for enabling `SBD_DELAY_START=yes` for the case of diskless sbd has been merged upstream:
https://github.com/ClusterLabs/sbd/pull/133

, so we could enable it here for diskless sbd as well. Probably it should belong to a separate PR/commit.

Reply:
OK, I will create another PR for this